### PR TITLE
CUDA 11.3 remove. New Stable version is 11.6

### PR DESCRIPTION
**Pull request by atalman**
CUDA 11.3 Removing.

Core PR: https://github.com/pytorch/pytorch/pull/84866
cc @malfet @ptrblck

### CODE REVIEW

**Review by datumbox — DISMISSED** [review]
The changes LGTM, thanks @atalman! Let's wait for the CI tests to be green.

BTW do you know if for v11.6 conda will install everything (including CUDA)? I tried this recently and led to issues on our Training Infra (AWS Cluster). v11.3 was actually the only compatible; I assume that you clipped support from core and we can no longer have it here right? I will let sixnines know but it would be good to give them a heads up to ensure that we have a proper CUDA support on the cluster.

**Comment by atalman** [comment]
> The changes LGTM, thanks @atalman!
> 
> BTW do you know if for v11.6 conda will install everything (including CUDA)? I tried this recently and led to issues on our Training Infra (AWS Cluster). v11.3 was actually the only compatible; I assume that you clipped support from core and we can no longer have it here right? I will let sixnines know but it would be good to give them a heads up to ensure that we have a proper CUDA support on the cluster.

This should install pytorch+cuda:
```
conda install pytorch torchvision torchaudio pytorch-cuda=11.6 -c pytorch-nightly -c nvidia
```

**Comment by datumbox** [comment]
@atalman It seems it works now.

Previously it didn't like the conflict between the pre-installed CUDA and the one that was brought by Conda. Now I started from a fresh env and everything worked.

**Review by seemethere — DISMISSED** [review]

**Comment by datumbox** [comment]
On Windows it seems we are [missing](https://app.circleci.com/pipelines/github/pytorch/vision/20072/workflows/7e7c4d42-b6f4-4de6-b73f-26c208cbf2e4/jobs/1630896) nvJPEG support:
```
_ test_decode_jpeg_cuda[False-ImageReadMode.UNCHANGED-grace_hopper_517x606.jpg] _
Traceback (most recent call last):
  File "C:\Users\circleci\project\test\test_image.py", line 366, in test_decode_jpeg_cuda
    img_nvjpeg = f(data, mode=mode, device="cuda")
  File "c:\users\circleci\project\torchvision\io\image.py", line 161, in decode_jpeg
    output = torch.ops.image.decode_jpeg_cuda(input, mode.value, device)
  File "C:\Users\circleci\project\env\lib\site-packages\torch\_ops.py", line 175, in __call__
    return self._op(*args, **kwargs or {})
RuntimeError: decode_jpeg_cuda: torchvision not compiled with nvJPEG support
```

Also again on windows GPU, we [fail](https://app.circleci.com/pipelines/github/pytorch/vision/20072/workflows/9cfd36cf-7135-4b92-aab8-0641b5bee5d6/jobs/1630866) with the `code exit status 127` problem again.

File: packaging/build_cmake.sh [hunk 1]
```diff
@@ -97,7 +97,7 @@ if [[ "$OSTYPE" == "msys" ]]; then
     "$script_dir/windows/internal/vc_env_helper.bat" "$script_dir/windows/internal/build_frcnn.bat" $PARALLELISM
     mv fasterrcnn_resnet50_fpn.pt Release
     cd Release
-    export PATH=$(cygpath -w "C:/Program Files (x86)/torchvision/bin"):$(cygpath -w $TORCH_PATH)/lib:$PATH
+    export PATH=$(cygpath -w "C:/Program Files/NVIDIA Corporation/NvToolsExt/bin/x64"):$(cygpath -w "C:/Program Files (x86)/torchvision/bin"):$(cygpath -w $TORCH_PATH)/lib:$PATH
```
Comment (atalman) [inline packaging/build_cmake.sh:100]: Please note failure in windows gpu job was cased by a missing dll that is located under:
```
C:/Program Files/NVIDIA Corporation/NvToolsExt/bin/x64
```

**Review by malfet — APPROVED** [review]
LGTM, though NVTX again should be part of the toolkit. Did we fail to install it to the script as well?

**Review by datumbox — APPROVED** [review]
LGTM, thanks!

**Comment by datumbox** [comment]
We might have a breakage linked to this PR. See https://github.com/pytorch/vision/issues/6589

@atalman @malfet  I'm not sure what's the best course of action here. Shall we revert to let more time investigate what's wrong with the CUDA kernels on Windows?

**Pull request merged**